x-pack/plugins/security_solution/cypress/integration/detection_rules/export_rule.spec.ts

```diff
@@ -86,4 +88,51 @@ describe('Export rules', () => {
       `Successfully exported ${expectedNumberCustomRulesToBeExported} of ${totalNumberOfRules} rules. Prebuilt rules were excluded from the resulting file.`
     );
   });
+
+  context('rules with exceptions', () => {
+    before(() => {
+      deleteExceptionList(getExceptionList().list_id, getExceptionList().namespace_type);
+    });
+
+    beforeEach(() => {
+      // create rule with exceptions
+      createExceptionList(getExceptionList(), getExceptionList().list_id).then((response) =>
+        createCustomRule(
```

Comment:
Why do we delete this exception list only once but create it before each test?
It looks like it will fail once we add the second test to this context.

Reply:
changed, thanks for the suggestion